tests/pipeline/test_composite_planner.py

```diff
@@ -59,7 +60,7 @@
 )
 groups_storage = get_storage(StorageKey.GROUPEDMESSAGES)
 groups_table = Table(
-    "groupedmessage_local",
+    "groupedmessage_dist",
```

Comment:
This should probably be similar to what you did in test_sessions_processing.py. i.e.`schema.get_table_name()` so that it continues working with the local tables.